code/game/gamemodes/clock_cult/clock_helpers/fabrication_helpers.dm

```diff
@@ -27,9 +27,6 @@
 /turf/closed/wall/mineral/cult/fabrication_vals(mob/living/user, obj/item/clockwork/replica_fabricator/fabricator, silent) //no metal
 	return list("operation_time" = 80, "new_obj_type" = /turf/closed/wall/clockwork, "power_cost" = POWER_WALL_TOTAL, "spawn_dir" = SOUTH)
 
-/turf/closed/wall/shuttle/fabrication_vals(mob/living/user, obj/item/clockwork/replica_fabricator/fabricator, silent) //two sheets of metal
```

Comment:
Shouldn't this be changed to /turf/closed/wall/mineral/titanium instead then?

Reply:
it's covered by an existing one; `/turf/closed/wall/mineral/fabrication_vals(mob/living/user, obj/item/clockwork/replica_fabricator/fabricator, silent)`

Reply:
Oh lovely, was looking for /titanium and missed it.